src/sprites/sprite.js

```diff
@@ -54,4 +54,27 @@ Sprite.prototype.createClone = function () {
     return newClone;
 };
 
+Sprite.prototype.export = function () {
+    var result = new Object();
+    var notSaved = ['clones', 'runtime'];
+    var x = null;
+    for (x in this) {
+        if (typeof this[x] === 'function') {
+            continue;
+            if (this.runtime.targets.testing == true) {
+                console.log('Ignoring ' + x);
```

Comment:
The VM should never write to the console directly. See: http://www.linfo.org/rule_of_silence.html